src/systems/animation.js

```diff
@@ -0,0 +1,31 @@
+var registerSystem = require('../core/system').registerSystem;
+var THREE = require('../lib/three');
+
+var camera = null;
+var scene = null;
+
+/**
+ * Animation system.
+ *
+ * Controls library-level animation events agnostic to implementations.
+ *
+ */
+module.exports.System = registerSystem('animation', {
+  init: function () {
+    this.sceneEl.addEventListener('camera-ready', function () {
+      scene = this.sceneEl.object3D;
+      camera = this.sceneEl.camera;
+    }, true);
+  },
+
+  /**
+   * Update animation mixers and shaders.
+   *
+   */
+  tick: function () {
+    if (scene && camera) {
+      THREE.glTFAnimator.update();

```

Comment:
Moved these here per @dmarcos 's suggestion -- I called it animation because I'd assume we'd want to use the regular animation mixer here per @donmccurdy 's comments.  

Not making any command decisions here, easy to rename to this to glTF-specific, or call it something relating to imported objects / models?  Or tack it into another system?